tools/monorepo-utils/src/workflow-profiler/commands/profile/index.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getWorkflowRunData,
+	getWorkflowData,
+	getRunJobData,
+	getCompiledJobData,
+} from '../../lib/data';
+import {
+	logWorkflowRunResults,
+	logJobResults,
+	logStepResults,
+} from '../../lib/log';
+import { Logger } from '../../../core/logger';
+
+const program = new Command( 'profile' )
+	.description( 'Profile GitHub workflows' )
+	.argument( '<start>', 'Start date in YYYY-MM-DD format' )
+	.argument( '<end>', 'End date in YYYY-MM-DD format' )
+	.argument(
+		'<id>',
+		'Workflow ID. The required workflow ids are 22745783, 5687250, 23271226, and 5461563. For the rest, use the `list` command to see workflow names and ids.'
```

Comment:
These IDs are going to be prone to drifting since they likely won't be updated here if we change the workflows. Also, I wonder, do these IDs ever change?